cortx-motr.spec.in

```diff
@@ -39,15 +40,19 @@
 %define  configure_cassandra_opts   --with-cassandra
 %endif
 
+%if %{with ivt}
+%define  configure_opts  %{configure_ut_opts} %{?configure_cassandra_opts}
+%else
 %if %{with ut}
 %define  configure_opts  %{configure_ut_opts} %{?configure_cassandra_opts}
 %else
 %define  configure_opts  %{configure_release_opts} %{?configure_cassandra_opts}
-%endif
+%endif #with ut
+%endif #with ivt
 
-Name:           @PACKAGE@
-Version:        @PACKAGE_VERSION@
-Release:        %{build_num}_@GIT_REV_ID_RPM@_%{kernel_ver}%{?dist}
+Name:           cortx-motr
+Version:        2.0.0
+Release:        %{build_num}_git1725ba7_%{kernel_ver}%{?dist}
```

Comment:
Please revert these changes.
They are replaced in the automake/autoconf step.

Reply:
Done 3ce805d